assets/main.js

```diff
@@ -151,6 +151,66 @@ window.onload = function () {
   });
 };
 
+var slideIndex = 0;
+showSlides();
+
+// Auto play slider on the Home page
+var autoPlaySlider = setInterval(function () {
+  showSlides(slideIndex += 1);
+}, 5000);
+
+// Toggle to next slide on the Home page
+function nextSlide() {
+  continueAutoPlay = false;
+  showSlides(slideIndex += 1);
+}
+
+// Toggle to selected slide on the Home page
+function currentSlide(n) {
+  clearInterval(autoPlaySlider);
+  showSlides(slideIndex = n);
+}
+
+// Toggle to previous slide on the Home page
+function prevSlide() {
+  clearInterval(autoPlaySlider);
+  showSlides(slideIndex -= 1);
+}
+
+// Toggle slides on the Home page
+function showSlides(n) {
+
+  var slides = document.getElementsByClassName("slidr");
```

Comment:
The code looks good 
Could there be a possibility of this being null?

Reply:
Thank you @badyokita, No there isn't as there will always be at least one slide in the slider (Since we aren't relying on third party dependencies or JS execution anymore, but rather standard html), but even if we didn't have one slide, I do an if(slides) check on line 192.